share/tools/gmt_getremote.sh

```diff
@@ -34,18 +35,41 @@ fi
 
 # Get current remote data server
 SERVER=$(gmt get GMT_DATA_SERVER)
+
+# The "gmt which" command
+gmtwhich="gmt which -Gu"
+#Uncomment the line below for a dry-run
+# gmtwhich="printf %s\n"
```

Comment:
You can uncomment this line to check which files will be downloaded without downloading them.